README.rst

```diff
@@ -10,395 +10,14 @@ powered by the BigQuery engine.
 BigQuery DataFrames is an open-source package. You can run
 ``pip install --upgrade bigframes`` to install the latest version.
 
+
 Documentation
 -------------
 
+* `Use BigQuery DataFrames <https://cloud.google.com/bigquery/docs/use-bigquery-dataframes>`_
```

Comment:
This is a broken link.

Reply:
I recommend the quickstart: https://cloud.google.com/bigquery/docs/dataframes-quickstart